SAPex/appsettings.json

```diff
@@ -6,5 +6,9 @@
       "Microsoft.Hosting.Lifetime": "Information"
     }
   },
-  "AllowedHosts": "*"
+  "AllowedHosts": "*",
+
+  "ConnectionStrings": {
+    "DefaultConnection": "Server=DESKTOP-4AVOV83\\SQLEXPRESS;Database=SAPExDb;Trusted_Connection=True;MultipleActiveResultSets=true"
```

Comment:
please create a new profile in SQL server management studio with mock crediantial. because this project will use other guys in your team. and they don't need to change the connection string. you and other guys should use one entering point to connect with it.